### PR TITLE
[stable/mm-te] deprecate MM-TE

### PR DESCRIPTION
**Pull request by cpanato**
#### What this PR does / why we need it:
deprecate MM-TE.
New chart repo: https://helm.mattermost.com

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/mattermost-team-edition/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 3.1.1
+version: 3.1.2
 appVersion: 5.9.0
```
Comment (davidkarlsen) [inline stable/mattermost-team-edition/Chart.yaml:5]: add `deprecated: true`as well

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/mattermost-team-edition/Chart.yaml (hunk 1 above)
Comment (cpanato) [reply stable/mattermost-team-edition/Chart.yaml:5]: thanks!

Comment (cpanato) [reply stable/mattermost-team-edition/Chart.yaml:5]: done

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**